imod_coupler/drivers/dfm_metamod/mapping.py

```diff
@@ -0,0 +1,622 @@
+import errno
+import os
+from pathlib import Path
+from typing import Any, Dict, Optional
+
+import numpy as np
+from numpy import float_, int_
+from numpy.typing import NDArray
+from scipy.sparse import csr_matrix, dia_matrix, diags
+
+from imod_coupler.utils import Operator, create_mapping
+
+
+class Mapping:
+    def __init__(self, coupling: Any, mf6_wrapper: Any, msw_wrapper: Any) -> None:
```

Comment:
as discussed, we need to think about making this class independent of these constructor arguments, because now it has become hard to set up tests for this class (they requiere a coupling object, a mf6_wrapper etc) 
In any case for now please provide more specific types (not " Any" ) for the mypy type checking. If everything is "Any" then type checking becomes not so usefull.